.gitmodules

```diff
@@ -17,5 +17,5 @@
 	path = boards/firechip/drivers/iceblk-driver
 	url = https://github.com/firesim/iceblk-driver.git
 [submodule "workloads/sha3"]
```

Comment:
For consistency, the submodule name should follow the path change.

Reply:
wow, git really doesn't fix that? I even used 'git mv' to do it. Whatever. It's fixed now.

Reply:
It's hard to say whether this omission is deliberate or not because renaming might break existing `.git/config` settings under the old name, although I have not checked.  But the discrepancy would be confusing to someone running `git config` expecting to use the new path.